### PR TITLE
Examples and links added

### PR DESCRIPTION
**Pull request by ajithkumar-maragathavel**
## Purpose of this pull request

This pull request (PR) adds request/response examples for _customAttributeMetadata_ and also provided devdocs links to products query page.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/graphql/custom-filters.html

### CODE REVIEW

File: src/guides/v2.3/graphql/custom-filters.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ As of Magento 2.3.4, the `filter` attribute of the [`products`]({{page.baseurl}}
 
 ## Prerequisites
 
-You have several options when enabling a custom attribute (or any attribute that is not listed by default in the `ProductAttributeFilterInput` object) for filtering. Navigate to the attribute's **Storefront Properties** page (**Stores** > Attributes > **Product** > <attribute name> > **Storefront Properties**) in the Admin, then perform one or both of the following actions:
+You have several options when enabling a custom attribute (or any attribute that is not listed by default in the `ProductAttributeFilterInput` object) for filtering. Navigate to the attribute's **Storefront Properties** page (**Stores** > Attributes > **Product** > Edit Any Attribute > **Storefront Properties**) in the Admin, then perform one or both of the following actions:
```
Comment (keharper) [inline src/guides/v2.3/graphql/custom-filters.md:10]: Revert this comment.

File: src/guides/v2.3/graphql/custom-filters.md [hunk 2]
```diff
@@ -25,13 +25,84 @@ The [`filter`]({{page.baseurl}}/graphql/queries/products.html#ProductFilterInput
 
 In this example, the custom attribute `volume` was assigned to the `bags` attribute group. Running the [`customAttributeMetadata` query]({{page.baseurl}}/graphql/queries/custom-attribute-metadata.html) on this custom attribute reveals that the `label` and `value` values for the attribute's options are as follows:
 
+**Endpoint:**
```
Comment (keharper) [inline src/guides/v2.3/graphql/custom-filters.md:28]: Remove the endpoint subsection

File: src/guides/v2.3/graphql/custom-filters.md [hunk 3]
```diff
@@ -25,13 +25,84 @@ The [`filter`]({{page.baseurl}}/graphql/queries/products.html#ProductFilterInput
...
+```
+
 `label` | `value`
 --- | ---
 `Large` | `216`
 `Medium` | `217`
 `Small` | `218`
 
-In this scenario, a `products` search filtered to return items where the `volume` attribute is set to `Large` would be similar to the following:
+In this scenario, a [`products`]({{page.baseurl}}/guides/v2.4/graphql/custom-filters.html) search filtered to return items where the `volume` attribute is set to `Large` would be similar to the following:
```
Comment (keharper) [inline src/guides/v2.3/graphql/custom-filters.md:99]: This link is invalid. Remove `guides/v2.4/` from the path. I think you're trying to link the `products` query, If so, the link would be `({{page.baseurl}}/graphql/queries/products.html)`

File: src/guides/v2.3/graphql/custom-filters.md [hunk 4]
```diff
@@ -25,13 +25,84 @@ The [`filter`]({{page.baseurl}}/graphql/queries/products.html#ProductFilterInput
...
+
+**Endpoint:**
```
Comment (keharper) [inline src/guides/v2.3/graphql/custom-filters.md:101]: Remove the endpoint subsection

**Review by keharper — CHANGES_REQUESTED** [review]

File: src/guides/v2.3/graphql/custom-filters.md (hunk 1 above)
Comment (ajithkumar-maragathavel) [reply src/guides/v2.3/graphql/custom-filters.md]: Updated @keharper

File: src/guides/v2.3/graphql/custom-filters.md (hunk 3 above)
Comment (ajithkumar-maragathavel) [reply src/guides/v2.3/graphql/custom-filters.md]: Link modified.

File: src/guides/v2.3/graphql/custom-filters.md (hunk 4 above)
Comment (ajithkumar-maragathavel) [reply src/guides/v2.3/graphql/custom-filters.md]: Removed.

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Pull request merged**